tests/src/Context/MinkContext.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace Drupal\joinup\Context;
+
+use Behat\Mink\Driver\GoutteDriver;
+use Drupal\DrupalExtension\Context\MinkContext as DrupalExtensionMinkContext;
+
+/**
+ * Provides step definitions for interacting with Mink.
+ */
+class MinkContext extends DrupalExtensionMinkContext {
+
+  /**
+   * Attach File to field.
+   *
+   * @param string $field
+   *   Field name.
+   * @param string $path
+   *   File path.
+   */
+  public function attachFileToField($field, $path) {
+    $field = $this->fixStepArgument($field);
+    if ($this->getMinkParameter('files_path')) {
+      $full_path = rtrim(realpath($this->getMinkParameter('files_path')), DIRECTORY_SEPARATOR)
+      . DIRECTORY_SEPARATOR
+        . $path;
+      if (is_file($full_path)) {
+        $path = $full_path;
+      }
+    }
+    if (!$this->getSession()->getDriver() instanceof GoutteDriver) {
```

Comment:
Only checking if the driver is not a `GoutteDriver` is not sufficient. This code will break when run using e.g. `PhantomJSDriver` which we use heavily on our development machines.